Build fails

On a clean system (enki not previously installed), `make` fails with the following error:
```
In file included from /home/damjan/devel/enki/enki/interactions/IRSensor.cpp:34:0:
/home/damjan/devel/enki/enki/interactions/IRSensor.h:37:33: fatal error: enki/PhysicalEngine.h: No such file or directory
compilation terminated.
```

Changing the capitalization on line 3 of the top-level CMakeLists.txt file from `project(Enki)` to `project(enki)` fixes the issue.

--- a/enki/CMakeLists.txt
+++ b/enki/CMakeLists.txt
@@ -1,4 +1,4 @@
-include_directories (${enki_SOURCE_DIR})
+include_directories (${PROJECT_SOURCE_DIR})
 
 if (CMAKE_HOST_UNIX)
 	add_definitions("-fPIC")

--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -1,5 +1,5 @@
-include_directories (${enki_SOURCE_DIR})
-link_directories (${enki_BINARY_DIR})
+include_directories (${PROJECT_SOURCE_DIR})
+link_directories (${PROJECT_BINARY_DIR})
 
 add_executable(enkiTest enkiTest.cpp)
 target_link_libraries(enkiTest enki)

--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -17,7 +17,7 @@ if (PYTHONLIBS_FOUND AND PYTHONINTERP_FOUND AND QT4_FOUND AND OPENGL_FOUND)
 	find_package(Boost COMPONENTS python)
 	if (Boost_FOUND)
 		message(STATUS "boost::python found, generating python bindings")
-		include_directories(${enki_SOURCE_DIR} ${PYTHON_INCLUDE_DIRS} ${Boost_INCLUDE_DIR})
+		include_directories(${PROJECT_SOURCE_DIR} ${PYTHON_INCLUDE_DIRS} ${Boost_INCLUDE_DIR})
 		python_add_module(pyenki enki.cpp)
 		target_link_libraries(pyenki enki enkiviewer ${QT_LIBRARIES} ${OPENGL_LIBRARIES} ${Boost_LIBRARIES} ${PYTHON_LIBRARIES})
 		# fix for old python_add_module

--- a/viewer/CMakeLists.txt
+++ b/viewer/CMakeLists.txt
@@ -45,9 +45,9 @@ if (QT4_FOUND AND OPENGL_FOUND)
 	
 	qt4_add_resources(viewer_lib_RCC_SRCS enki-viewer-textures.qrc)
 	
-	include_directories (${enki_SOURCE_DIR} ${OPENGL_INCLUDE_DIR} ${CMAKE_BINARY_DIR}/viewer)
+	include_directories (${PROJECT_SOURCE_DIR} ${OPENGL_INCLUDE_DIR} ${CMAKE_BINARY_DIR}/viewer)
 	
-	link_directories (${enki_BINARY_DIR})
+	link_directories (${PROJECT_BINARY_DIR})
 	
 	add_library (enkiviewer ${viewer_lib_SRCS} ${viewer_lib_RCC_SRCS})
 	set_target_properties(enkiviewer PROPERTIES VERSION ${LIB_VERSION_STRING} 

